app/assets/javascripts/books.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I know it was generated together with model, but we dont need these coffescripts files yet.  Plz remove them since we done want to have them committed